### PR TITLE
stern 1.0.1 (new formula)

### PR DESCRIPTION
**Pull request by mgoodness**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

New formula for a Kubernetes log-tailer from Wercker. The test (`stern --version`) isn't great, so I'm open to other suggestions.

### CODE REVIEW

File: Formula/stern.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
+class Stern < Formula
+  desc "Tail multiple Kubernetes pods & their containers."
+  homepage "https://github.com/wercker/stern"
+  url "https://github.com/wercker/stern.git",
+    :tag => "1.0.1",
+    :revision => "3c8e6184b7a0a0cbc8dc78f219be0a49a007e861"
```
Comment (MikeMcQuaid) [inline Formula/stern.rb]: This doesn't seem to use submodules so we prefer https://github.com/wercker/stern/archive/1.0.1.tar.gz

File: Formula/stern.rb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  head "https://github.com/wercker/stern",
+    :shallow => false
+
+  depends_on "go" => :build
+  depends_on "govendor" => :build
+
+  def install
+    contents = Dir["{*,.git,.gitignore}"]
+    gopath = buildpath/"gopath"
+    (gopath/"src/github.com/wercker/stern").install contents
```
Comment (MikeMcQuaid) [inline Formula/stern.rb]: Can you symlink rather than `.install` here?

**Review by MikeMcQuaid — CHANGES_REQUESTED** [review]

File: Formula/stern.rb (hunk 2 above)
Comment (mgoodness) [inline Formula/stern.rb]: Apparently not. I'm unable to pull dependencies via `govendor sync` if I symlink. Full disclosure: my golang-fu is not strong.

**Review by MikeMcQuaid — APPROVED** [review]

**Comment by MikeMcQuaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request closed without merging**